conf/application.conf

```diff
@@ -70,21 +70,14 @@ assets {
     url = "http://localhost:9032/assets/"
 }
 
-Prod {
-    assets {
-        url="/assets/"
-        version="3.2.2"
-        minified=true
-    }
-}
-Test {
-    assets {
-        url="/assets/"
-        version="3.2.2"
-        url = "http://localhost:9032/assets/"
-    }
+
+assets {
+    url="/assets/"
+    version="3.2.2"
+    url = "http://localhost:9032/assets/"
```

Comment:
If there is already an assets section is this one necessary given that the stuff that is set in here is already set in the first section. I can't see if there is a reason for this unless there is a getOrElse that sets default values for the assets stuff that isn't set within here